dev-python/virtualenvwrapper/virtualenvwrapper-4.8.4-r3.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_SINGLE_IMPL=1
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{10..11} )
+
+inherit distutils-r1 pypi
+
+DESCRIPTION="Set of extensions to Ian Bicking's virtualenv tool"
+HOMEPAGE="https://github.com/python-virtualenvwrapper/virtualenvwrapper
+	https://pypi.org/project/virtualenvwrapper/"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+
+RDEPEND="
+	$(python_gen_cond_dep '
+		dev-python/virtualenv[${PYTHON_USEDEP}]
+		dev-python/stevedore[${PYTHON_USEDEP}]
+		dev-python/virtualenv-clone[${PYTHON_USEDEP}]
+	')"
+BDEPEND="
+	${DEPEND}
+	$(python_gen_cond_dep '
+		dev-python/pbr[${PYTHON_USEDEP}]
+	')"
+
+PATCHES=(
+	"${FILESDIR}/${P}-adjust-test-error-message.patch"
+	"${FILESDIR}/${P}-dont-expect-easy_install.patch"
+	"${FILESDIR}/${P}-fix-wipeenv.patch"
+	"${FILESDIR}/${P}-remove-nositepackages-feature.patch"
+	"${FILESDIR}/${P}-remove-relocatable-tests.patch"
+)
+
+src_prepare() {
+	default
+	sed -i -e 's/egrep/grep -E/' "${S}/virtualenvwrapper.sh" || die
```

Comment:
Was this change done upstream?

Reply:
https://github.com/python-virtualenvwrapper/virtualenvwrapper/commit/1a7bec7500ce2af4648a70b9f301810d4afca9d3

It has been done in the next release. sed was used before so I opted to keep it as is rather than backporting it.

Reply:
In regards to the previous point on a snapshot. If I were to make a snapshot then this change would most likely included, making the line redundant.